fem/fespace.hpp

```diff
@@ -254,10 +276,18 @@ class FiniteElementSpace
    Table var_edge_dofs;
    Table var_face_dofs; ///< NOTE: also used for spaces with mixed faces
 
+   /// Bit-mask representing a set of orders needed by an edge/face.
+   typedef std::uint64_t VarOrderBits;
+   static constexpr int MaxVarOrder = 8*sizeof(VarOrderBits) - 1;
```

Comment:
Doesn't this always evaluate to 63 since `uint64_t` is fixed at 8 bytes?

Reply:
Yes, but the expression is more informative than `= 63`. This was already in this file before this PR. I simply moved it higher in the file.

Reply:
It is alright.. I think I understand why `MaxVarOrder` is calculated the way it is.